client/state/plans/test/selectors.js

```diff
@@ -84,12 +87,43 @@ describe( 'selectors', () => {
 			const price = getPlanRawPrice( state, 1003, true );
 			expect( price ).to.eql( 0 );
 		} );
+		test( 'should return monthly price plan object when term is biennial', () => {
+			const state = deepFreeze( {
+				plans: {
+					items: [
+						{
+							product_id: 1029,
+							product_slug: 'personal-bundle-2y',
+							raw_price: 240,
+						},
+					],
+				},
+			} );
+			const price = getPlanRawPrice( state, 1029, true );
```

Comment:
how are we supposed to know that this is a biennial product? is it because the slug ends in `-2y`? (that's a total guess)

Reply:
In other PR I added some meta attributes to our plan constants, so each plan is marked with proper term like in here: https://github.com/Automattic/wp-calypso/blob/master/client/lib/plans/constants.js#L435

Reply:
oh, so this depends on the plan constants...

Reply:
yeah so it's awkward to have the test functions depend on production code. is there a way that we can fake the entire set of data in the test file or is it too tightly coupled to really be testable that way?